### PR TITLE
LPD-39372 The "Make a copy" action should be "Copy to" action

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
> [!Note]
> There are integration tests for this action, but I added a playwright one to test the new modal due to the copy to action

## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPD-39372)

### Changes

Chage the copy action for the folders in display pages - due to requirements

## Test scenario 

1. Create two folders
2. Go to the kebab menu of one of them
3. Click on copy to
4. Select the other folder to copy into it
5. Check that the copy has been made
6. Try it also using the page management action

[Screencast from 2024-10-17 00-19-50.webm](https://github.com/user-attachments/assets/5dd8cd85-01da-4f7f-9f21-07c6f3ff33f1)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by BarbaraCabrera** [comment]
ci:test:page-management

File: modules/apps/layout/layout-page-template-admin-web/src/main/resources/META-INF/resources/js/propsTransformers/LayoutPageTemplateCollectionPropsTransformer.js [hunk 1]
```diff
@@ -14,6 +14,41 @@ import {
...
+						selectedItem.resourceid,
+				});
+
+				submitForm(form);
+			},
+			selectEventName: 'selectFolder',
+			size: 'md',
+			title: sub(
+				Liferay.Language.get('copy-x-to'),
+				`"${layoutPageTemplateCollectionName}"`
```
Comment (victorg1991) [inline modules/apps/layout/layout-page-template-admin-web/src/main/resources/META-INF/resources/js/propsTransformers/LayoutPageTemplateCollectionPropsTransformer.js:46]: You can pass the variable directly without embedding it in a string :D

File: modules/apps/layout/layout-page-template-admin-web/src/main/resources/META-INF/resources/view_display_pages.jsp [hunk 2]
```diff
@@ -146,6 +146,17 @@ DisplayPageManagementToolbarDisplayContext displayPageManagementToolbarDisplayCo
...
 
+<portlet:actionURL name="/layout_page_template_admin/copy_layout_page_template_entries_and_layout_page_template_collections" var="copyEntriesURL">
+	<portlet:param name="redirect" value="<%= currentURL %>" />
+</portlet:actionURL>
+
+<aui:form action="<%= copyEntriesURL %>" name="actionEntriesFm">
+	<aui:input name="layoutPageTemplateCollectionsIds" type="hidden" />
+	<aui:input name="layoutPageTemplateEntriesIds" type="hidden" />
+	<aui:input name="layoutParentPageTemplateCollectionId" type="hidden" />
+	<aui:input name="copypermisisons" type="hidden" />
```
Comment (victorg1991) [inline modules/apps/layout/layout-page-template-admin-web/src/main/resources/META-INF/resources/view_display_pages.jsp:157]: Here it should be `copyPermissions` right?

Comment (BarbaraCabrera) [reply modules/apps/layout/layout-page-template-admin-web/src/main/resources/META-INF/resources/view_display_pages.jsp:157]: Oui 🇫🇷

File: modules/apps/layout/layout-page-template-admin-web/src/main/resources/META-INF/resources/js/propsTransformers/LayoutPageTemplateCollectionPropsTransformer.js (hunk 1 above)
Comment (BarbaraCabrera) [reply modules/apps/layout/layout-page-template-admin-web/src/main/resources/META-INF/resources/js/propsTransformers/LayoutPageTemplateCollectionPropsTransformer.js:46]: Arigato 🇯🇵

**Review by lfbesada — COMMENTED** [review]
@BarbaraCabrera It works great! :clap: 
I've added a few SF commits, please see: https://github.com/liferay-echo/liferay-portal/pull/16434
Regards

**Pull request closed without merging**